### PR TITLE
Package ogg.0.7.4

### PR DESCRIPTION
**Pull request by toots**
### `ogg.0.7.4`
Bindings to libogg

---
* Homepage: https://github.com/savonet/ocaml-ogg
* Source repo: git+https://github.com/savonet/ocaml-ogg.git
* Bug tracker: https://github.com/savonet/ocaml-ogg/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/ogg/ogg.0.7.4/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libogg"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```
Comment (raphael-proust) [inline packages/ogg/ogg.0.7.4/opam:6]:
```suggestion
license: "GPL-2.0-only"
```

or 

```suggestion
license: "GPL-2.0-or-later"
```

Comment (toots) [reply packages/ogg/ogg.0.7.4/opam]: Thanks! It was actually the wrong license.. Fix pushed upstream.

File: packages/ogg/ogg.0.7.4/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libogg"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exceptio"
```
Comment (mseri) [inline packages/ogg/ogg.0.7.4/opam:6]:
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```
I'll merge this before merging the PR, after the CI has finished

Comment (toots) [reply packages/ogg/ogg.0.7.4/opam:6]: Thanks, The typo is fixed upstream.

**Pull request merged**